File: Tareas/Unidad2/a2253332351_practica01/08_PrismaPentagonal.java
import java.util.Scanner;

public class PrismaPentagonal {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double perimetro, apotema, altura, aTotal, aLado, aBase, volumen;

            System.out.println("Ingresa el perimetro de la Base: ");
            perimetro = entrada.nextDouble();
            System.out.println("Ingresa la apotema: ");
            apotema = entrada.nextDouble();
            System.out.println("Ingresa la altura: ");
            altura = entrada.nextDouble();

            aBase = (perimetro * apotema) / 2;
            aLado = perimetro * altura;
            aTotal = (2 * aBase) + aLado;
            volumen = aBase * altura;

            System.out.println("El área de la base es: " + aBase);
            System.out.println("El área lateral es: " + aLado);
            System.out.println("El área total es: " + aTotal);
            System.out.println("El volumen del prisma pentagonal es: " + volumen);
      }
}

File: Tareas/Unidad2/a2253332351_practica01/08_PrismaPentagonal.py
# PrismaPentagonal.py

perimetro = float(input("Ingresa el perimetro de la Base: "))
apotema = float(input("Ingresa la apotema: "))
altura = float(input("Ingresa la altura: "))

aBase = (perimetro * apotema) / 2
aLado = perimetro * altura
aTotal = (2 * aBase) + aLado
volumen = aBase * altura

print(f"El área de la base es: {aBase}")
print(f"El área lateral es: {aLado}")
print(f"El área total es: {aTotal}")
print(f"El volumen del prisma pentagonal es: {volumen}")
